finder-app/finder-test.sh

```diff
@@ -24,7 +24,7 @@ else
 	WRITESTR=$2
 fi
 
-MATCHSTR="The number of files are ${NUMFILES} and the number of matching lines are ${NUMFILES}"
+MATCHSTR="The number of files is ${NUMFILES} and the number of matching lines is ${NUMFILES}"
```

Comment:
@georgehodgkins I'm curious why the plural to singular change here.  Do we have instructions wrong somewhere?  Step 11 of assignment 1 instructions looks correct to me.